### PR TITLE
Add conditions for cronjobs

### PR DESCRIPTION
**Pull request by kajinamit**

### CODE REVIEW

**Comment by kajinamit** [comment]
I'm submitting this based on the discussion in https://github.com/openstack-k8s-operators/keystone-operator/pull/207

File: modules/common/condition/conditions.go [hunk 1]
```diff
@@ -210,4 +210,19 @@ const (
...
 	NetworkAttachmentsReadyErrorMessage = "NetworkAttachments error occured %s"
+
+	//
+	// CronjobReady condition messages
+	//
+	// CronjobReadyInitMessage
+	CronjobReadyInitMessage = "Cronjob not started"
+
+	// CronjobReadyMessage
+	CronjobReadyMessage = "Cronnjob completed"
```
Comment (stuggi) [inline modules/common/condition/conditions.go:221]: nit Cronn -> Cron

**Review by stuggi — APPROVED** [review]
/lgtm

File: modules/common/condition/conditions.go (hunk 1 above)
Comment (kajinamit) [reply modules/common/condition/conditions.go]: Fixed. Good catch !

**Review by kajinamit — COMMENTED** [review]
Oops. I forgot adding condition name. Will fix it by https://github.com/openstack-k8s-operators/lib-common/pull/216 .

**Pull request merged**